TestProject_Calc/CalculatorPage.cs

```diff
@@ -1,7 +1,184 @@
-﻿namespace TestProject_Calc
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using System.Reflection;
+
+namespace TestProject_Calc
 {
-    public class CalculatorPage()
+    public class CalculatorPage(ChromeDriver driver)
     {
         public static string URL = "http://localhost:5000/Calculator";
+        private IWebElement DepositAmmountField => driver.FindElement(By.XPath("//input[@id='amount']"));
+        private IWebElement RateOfInterestField => driver.FindElement(By.XPath("//input[@id='percent']"));
+        private IWebElement InvestmentTermField => driver.FindElement(By.XPath("//input[@id='term']"));
+        private IWebElement StartDateDay => driver.FindElement(By.XPath("//select[@id='day']"));
+        private IWebElement StartDateMonth => driver.FindElement(By.XPath("//select[@id='month']"));
+        public IWebElement StartDateYear => driver.FindElement(By.XPath("//select[@id='year']"));
+        private IWebElement RadioButton360 => driver.FindElement(By.XPath("//input[@id='finYear360']"));
+        private IWebElement RadioButton365 => driver.FindElement(By.XPath("//input[@id='finYear365']"));
+        public IWebElement CalculateButton => driver.FindElement(By.XPath("//button[@id='calculateBtn']"));
+        private IWebElement InteresetEarnedField => driver.FindElement(By.XPath("//input[@id='interest']"));
+        private IWebElement IncomeField => driver.FindElement(By.XPath("//input[@id='income']"));
+        private IWebElement EndDateField => driver.FindElement(By.XPath("//input[@id='endDate']"));
+        private PropertyInfo[] calculatorProperty = typeof(CalculatorValues).GetProperties();
+        public void InsertDepositAmmount(string depositValue) => DepositAmmountField.SendKeys(depositValue);
+        public void InsertRateOfInterest(string rateValue) => RateOfInterestField.SendKeys(rateValue);
+        public void InsertInvestmentTerm(string term) => InvestmentTermField.SendKeys(term);
+        public void InsertStartDateDay(string day) => new SelectElement(StartDateDay).SelectByValue(day);
+        public void InsertStartDateMonth(string month) => StartDateMonth.SendKeys(month);
+        public void InsertStartDateYear(string year) => StartDateYear.SendKeys(year);
+        public FinancialYearInput? CheckFinancialYear()
```

Comment:
Suggestion: add a line between methods or multi-line properties:
```suggestion
        public void InsertStartDateYear(string year) => StartDateYear.SendKeys(year);

         public FinancialYearInput? CheckFinancialYear()
```